platform/kube/controller.go

```diff
@@ -391,6 +411,7 @@ func (c *Controller) GetIstioServiceAccounts(hostname string, ports []string) []
 }
 
 func generateServiceAccountID(sa string, ns string, domain string) string {
+	fmt.Printf("%v://%v/ns/%v/sa/%v\n", uriScheme, domain, ns, sa)
```

Comment:
Service accounts in K8s doesn't have a scheme, it's just a yaml file.
The K8s API server has a scheme to indicate service account, such as:
v1/namespaces/foo/serivceaccounts/bar -- it have domain
So we plan to align with that, having the URI-type scheme like:
spiffe://domain/namespaces/foo/serviceaccounts/bar